resources/assets/js/src/services/firebase.ts

```diff
@@ -0,0 +1,22 @@
+import * as firebase from "firebase";
+
+const config = {
+    apiKey: process.env.MIX_FIREBASE_API_KEY,
+    authDomain: process.env.MIX_FIREBASE_AUTH_DOMAIN,
+    databaseURL: process.env.MIX_FIREBASE_DATABASE_URL,
+    messagingSenderId: process.env.MIX_FIREBASE_MESSAGING_SENDER_ID,
+    projectId: process.env.MIX_FIREBASE_PROJECT_ID,
+    storageBucket: process.env.MIX_FIREBASE_STORAGE_BUCKET,
+};
+
+try {
```

Comment:
you can leave it without  try catch statement, error message would appear anyways. Right solution would be showing of error screen to inform user that error occurred.

Reply:
I can't, this is only working solution to initialize firebase from multiple components (without try block, it will fail even with successful initialization because it will create new instance). Also user shouldn't be informed about that, the console log is there only for future development.